econpizza/__init__.py

```diff
@@ -24,6 +25,20 @@
 # create local alias
 copy = deepcopy
 
+if config.enable_jax_persistent_cache == True:
```

Comment:
I think these two `if` loops should be a function that gets called at the top of each solver. This way, any change in config would take effect even after the package is already loaded.

Reply:
Good point!